lint-staged.config.js

```diff
@@ -0,0 +1,11 @@
+const jsRule = [
+	'prettier --use-tabs --single-quote --trailing-comma=all --write',
+	'git add'
+];
+
+module.exports = {
+	'src/**/*.js': jsRule,
+	'test/index.test.js': jsRule,
```

Comment:
those rules for `test` dir probably could be rewritten to "every .js file in test but not in test/fixtures" but I always forgot how to do that with glob in combination with `**` :s

Reply:
I don't think it would be bad to include the fixtures, the only thing we don't want is the outputs

Reply:
and I suppose we could use the ignore option for that?